crates/red_knot/src/module.rs

```diff
@@ -12,6 +12,13 @@ use crate::files::FileId;
 use crate::semantic::Dependency;
 use crate::FxDashMap;
 
+// The file path here is hardcoded in this crate's `build.rs` script.
+// Luckily this crate will fail to build if this file isn't available at build time.
+//
+// Allow dead code for now; actually resolving modules to vendored typeshed stubs will come soon!
+#[allow(dead_code)]
+const TYPESHED_ZIP_BYTES: &[u8] = include_bytes!("../vendor/zipped_typeshed.zip");
```

Comment:
I recommend to move the const to the `tests` module for now so that you don't need the `dead_code` suppression